core/tests/test_utils.py

```diff
@@ -658,9 +733,16 @@ def get_json(self, url, params=None, expected_status_int=200):
         expect_errors = False
         if expected_status_int >= 400:
             expect_errors = True
-        json_response = self.testapp.get(
-            url, params, expect_errors=expect_errors,
-            status=expected_status_int)
+
+        # This swap is required to ensure that the templates are fetched from
+        # source directory instead of webpack_bundles since webpack_bundles
+        # is only produced after webpack compilation which is not performed
+        # during backend tests.
+        with self.swap(
+            jinja2.environment.Environment, 'get_template', mock_get_template):
+            json_response = self.testapp.get(
+                url, params, expect_errors=expect_errors,
+                status=expected_status_int)
```

Comment:
Do we really need to swap for JSON response?

Reply:
Yes, my bad. It is not required here. Removed.